util/locale/locale.go

```diff
@@ -22,43 +23,66 @@ var (
 	Localizer *i18n.Localizer
 )
 
+// Init initializes the localization bundle and loads all JSON message files.
 func Init() error {
 	Bundle = i18n.NewBundle(language.English)
-	Bundle.RegisterUnmarshalFunc("toml", toml.Unmarshal)
+	Bundle.RegisterUnmarshalFunc("json", json.Unmarshal)
 
 	dir, err := fs.ReadDir(assets.I18n, ".")
 	if err != nil {
 		panic(err)
 	}
 
+	// Iterate over each file and process only .json files
 	for _, d := range dir {
-		var data map[string]map[string]map[string]any
-		if _, err := toml.DecodeFS(assets.I18n, d.Name(), &data); err != nil {
-			return fmt.Errorf("loading locales failed: %w", err)
+		if filepath.Ext(d.Name()) != ".json" {
+			continue
 		}
 
-		// load sessions.csv only
-		if sessions := data["sessions"]; sessions != nil && len(sessions["csv"]) != 0 {
-			b, err := toml.Marshal(map[string]any{
-				"sessions": map[string]any{
-					"csv": sessions["csv"],
-				},
-			})
-			if err != nil {
-				return fmt.Errorf("marshal session.csv failed: %w", err)
-			}
+		b, err := fs.ReadFile(assets.I18n, d.Name())
+		if err != nil {
+			return fmt.Errorf("reading locale file %s failed: %w", d.Name(), err)
+		}
+
+		// Unmarshal into generic structure to inspect sessions field
+		var raw map[string]interface{}
+		if err := json.Unmarshal(b, &raw); err != nil {
+			return fmt.Errorf("unmarshal locale file %s failed: %w", d.Name(), err)
+		}
 
-			if _, err := Bundle.ParseMessageFileBytes(b, d.Name()); err != nil {
-				return fmt.Errorf("loading locales failed: %w", err)
+		// Session-specific logic: load only sessions.csv if present
```

Comment:
@andig how do you feel about dropping translation logic in go all together? Currently nearly all translation is done in UI. Only `sessions.csv.*` is used in go (`evcc configure` does it's own thing). We could simply hard code the csv column heads with the English terms and remove all of this code.

Reply:
> Update: ich sehe den Kontext. Warum sollen wir das hier nicht auf Basis json beibehalten?

Können wir gerne beibehalten. Ich hab die Frage gestellt, weil mir die Ratio von notwendigem Go-Init-Code zu den nur 14 Spaltentiteln etwas off erscheint. Mir sind zugegebenermaßen aber auch die Spaltentitel relativ egal weil ich nie CSV Exporte mache und ich auch mit Englisch kein Problem habe 😂